src/platform/Linux/CHIPPlatformConfig.h

```diff
@@ -64,6 +64,9 @@ using CHIP_CONFIG_PERSISTED_STORAGE_KEY_TYPE = const char *;
 #define CHIP_CONFIG_BDX_MAX_NUM_TRANSFERS 1
 #endif // CHIP_CONFIG_BDX_MAX_NUM_TRANSFERS
 
+// Enable subscription persistence and resumption for CI
+#define CHIP_CONFIG_PERSIST_SUBSCRIPTIONS 1
```

Comment:
This was supposed to be DISABLED by default (except on 1 platform, and Darwin is 1 platform) until spec work made this behavior well-defined at the client-level